pkgs/tools/filesystems/darling-dmg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchFromGitHub, lib, cmake, fuse, zlib, bzip2, openssl, libxml2, icu } :
+
+stdenv.mkDerivation rec {
+  name = "darling-dmg";
```

Comment:
One more nitpick: This should have a version or a timestamp to make it updateable via nix-env.